src/sentry/api/endpoints/organization_events.py

```diff
@@ -43,6 +43,9 @@
     DiscoverSavedQueryTypes.TRANSACTION_LIKE: get_dataset("transactions"),
     DiscoverSavedQueryTypes.ERROR_EVENTS: get_dataset("errors"),
 }
+# TODO: Adjust this once we make a decision in the DACI for global views restriction
+# Do not add more referres to this list as it is a temporary solution
+GLOBAL_VIEW_WHITELIST = ("api.issues.issue_events",)
```

Comment:
nit: I believe the terminology we use for this type of thing should be `allowlist`
https://develop.sentry.dev/development/inclusive-language/#whitelistblacklist